rokka/client.go

```diff
@@ -89,6 +91,10 @@ func (c *Client) Call(req *http.Request, v interface{}) error {
 
 	decoder := json.NewDecoder(resp.Body)
 
+	if responseHandler != nil {
+		return responseHandler(v, resp)
+	}
```

Comment:
the decoder we don't need at this point, so could be moved below this call.
What I think though is that the status code error handling should be done regardless of the response handler.